app/services/hex/comparer.rb

```diff
@@ -0,0 +1,50 @@
+module Hex
+  class Comparer
+    RGB_REGEX = /^#?(?<r>..)(?<g>..)(?<b>..)$/.freeze
+    ACCENT_MODIFIERS = [1.14, 1.08, 1.06, 0.96, 0.9, 0.8, 0.7, 0.6].freeze
+
+    def initialize(hexes)
+      @hexes = hexes.sort
+    end
```

Comment:
I removed the `amount` argument because we aren't using it anywhere in the code. Instead, we've been passing the amount argument to the specific methods themselves. In other words, we do:

```ruby
Hex::Comparer.new([enriched_colors[:bg], enriched_colors[:text]]).brightness(adjustment)
```

Instead of:
```ruby
Hex::Comparer.new([enriched_colors[:bg], enriched_colors[:text]], adjustment).brightness
```